.flake8

```diff
@@ -4,7 +4,7 @@ max-complexity = 10
 statistics = False
 # black compatibility
 max-line-length = 88
-extend-ignore = E203, E501
+extend-ignore = E203, E501, W293
```

Comment:
wdyt, would it make sense to ignore this error for one file only using [`per-file-ignore` flake configuration option](https://flake8.pycqa.org/en/latest/user/options.html#cmdoption-flake8-per-file-ignores)? it would also allow you to stop excluding files for flake8 pre-commit hook